diff --git a/dist/cjs/index.d.ts b/dist/cjs/index.d.ts
index 48de09cf75d55bb02a47986a388add16e174e8c3..44dcc3fe1b1797fd7aa9ff4915d28764397088bb 100644
--- a/dist/cjs/index.d.ts
+++ b/dist/cjs/index.d.ts
@@ -3,4 +3,4 @@ interface PackageManager {
     version: string;
 }
 declare const _default: () => PackageManager | undefined;
-export default _default;
+export = _default;
diff --git a/dist/cjs/index.js b/dist/cjs/index.js
index fb78983533ee507bf4475aea73c1fef53a37c385..54894bbae230b582af4925fd1403c1c13f11ee39 100644
--- a/dist/cjs/index.js
+++ b/dist/cjs/index.js
@@ -1,6 +1,5 @@
 "use strict";
-Object.defineProperty(exports, "__esModule", { value: true });
-exports.default = () => {
+module.exports = () => {
     const userAgent = process.env.npm_config_user_agent;
     if (!userAgent)
         return;
diff --git a/dist/esm/package.json b/dist/esm/package.json
new file mode 100644
index 0000000000000000000000000000000000000000..089153bcb5adf57dc25f206a9dad3114c9cff80d
--- /dev/null
+++ b/dist/esm/package.json
@@ -0,0 +1 @@
+{"type":"module"}